pengwin-setup.d/uninstall/nim.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+source $(dirname "$0")/uninstall-common.sh
+
+function main()
+{
+
+local nim_conf="/etc/profile.d/nim.sh"
```

Comment:
this doesn't seem to be used here, probably worth deleting this line

Reply:
Which line? I was just trying to track other uninstall scripts as closely as possible.

Reply:
the declaration of `nim_conf` then not using the variable in the script. if there's any examples of this in other uninstall scripts that'll be mistakes on my part i need to address :sweat_smile: actually gonna check over them now see if i've done that anywhere

Reply:
I based it on the Go script. https://github.com/WhitewaterFoundry/pengwin-setup/blob/master/pengwin-setup.d/uninstall/go.sh#L8